web/src/components/PageHeader/PageHeader.scss

```diff
@@ -1,7 +1,8 @@
 @use "src/general.scss";
 
 .page-header {
-  margin: 0.5rem;
+  padding-top: 2rem;
+  margin: 1rem;
```

Comment:
This fixes #554 by pushing the top of the page down, needs to be tested in the app as the issue we were having is entirely related to mobile safari, but this change will effect the app as well

I've only tested with safari thus far as I can locally host a webserver, but I can't advertise that webserver to the app afaik

Reply:
![image](https://github.com/micro-nova/AmpliPi/assets/114415937/e202862c-f7fb-4fc4-9a51-41cf46962ee4)

I've deployed to Lamplipi and it does not negatively effect the mobile layout

Reply:
Note that this is on an iPhone 8, the big gap at the top of the screen between the time and battery life info isn't part of the screen, but is where the camera and phone speaker go

Reply:
The true root of this issue was the List components that were custom built having weird styling that pushed stuff up for some reason

I've undone these changes and have not regained the problem they aimed to fix, so :+1: